project/script_IR.tcl

```diff
@@ -29,8 +29,8 @@ create_clock -period 360MHz -name fast_clock
 set nProc [exec nproc]
 csim_design -mflags "-j$nProc" -argv "--link,6 --dtcSplit,0 --tkNonant,4"
 csynth_design 
-# possible options -trace_level all -rtl verilog -verbose 
+# # possible options -trace_level all -rtl verilog -verbose 
 cosim_design  -argv "--link,6 --dtcSplit,0 --tkNonant,4"
-# possible options  -flow syn, -flow impl
+# # possible options  -flow syn, -flow impl
```

Comment:
Could you please undo these changes, since they don't affect anything?

Reply:
Sorry .. last commit was supposed to un-do that instead I commented them out a second time!

Reply:
Fixed in latest commit : 1055d5ee68c7c31bf35a9d2d940b5a9668b82483

Reply:
That did it!